app/src/main/java/net/programmierecke/radiodroid2/CountryCodeDictionary.java

```diff
@@ -22,30 +21,16 @@
 
 public class CountryCodeDictionary {
     private static final CountryCodeDictionary ourInstance = new CountryCodeDictionary();
-
-    public static CountryCodeDictionary getInstance() {
-        return ourInstance;
-    }
+    private Map<String, String> countryToCode = new HashMap<>();
```

Comment:
While code reformatting is something which should have been done at some time point even though it screws with `git blame`, I don't think moving code around is something we want. I'd like not to sort the code automatically.

Reply:
That sentiment does make sense, but if it's something that should've been done at some point, I don't see why we can't do it right now to just get it over with. It usually makes the code more readable as well.

I can undo the codestyle adjustment if you want it the way it was before, though.

Reply:
> While code reformatting is something which should have been done at some time point even though it screws with `git blame`.

Uh, this sentence had a stroke, sorry. I meant that reformatting as in changing whitespaces, curly braces and so on should be done at some point, so it's ok to do it now.   

While the sorting of methods and fields is in many cases left to the developers and I'd like to keep it that way.

Reply:
Alright, I'll get rid of the sorting but leave the reformatting.